.travis.yml

```diff
@@ -16,6 +16,12 @@ matrix:
     script:
       - sh ci/miri.sh
 
+  - env: CONST_OFFSET
+    rust: nightly
+    os: linux
+    script:
+      - cargo test --verbose --features unstable_const --lib
```

Comment:
Why "--lib"?

Reply:
I originally added it so that the doctests won't run with this feature, because then they'll fail because they do not specify the "feature" directives.

I guess I can remove this now that the macro is no longer forcing `const`.

Reply:
We can't remove this actually.

Reply:
Then please add a comment saying that this is because of the doctests not setting the feature flags.